awesome/keys.lua

```diff
@@ -157,35 +157,35 @@ keys.globalkeys = gears.table.join(
    -- Brightness
    awful.key({}, "XF86MonBrightnessUp",
       function()
-         awful.spawn("xbacklight -inc 10", false)
+         awful.spawn("brightnessctl s 5%+", false)
       end,
-      {description = "+10%", group = "hotkeys"}
+      {description = "+5%", group = "hotkeys"}
    ),
    awful.key({}, "XF86MonBrightnessDown",
       function()
-         awful.spawn("xbacklight -dec 10", false)
+         awful.spawn("brightnessctl s 5%-", false)
       end,
-      {description = "-10%", group = "hotkeys"}
+      {description = "-5%", group = "hotkeys"}
    ),
 
    -- ALSA volume control
    awful.key({}, "XF86AudioRaiseVolume",
       function()
-         awful.spawn("amixer -D pulse sset Master 5%+", false)
+         awful.spawn("amixer set Master 5%+", false)
          awesome.emit_signal("volume_change")
       end,
       {description = "volume up", group = "hotkeys"}
    ),
    awful.key({}, "XF86AudioLowerVolume",
       function()
-         awful.spawn("amixer -D pulse sset Master 5%-", false)
+         awful.spawn("amixer set Master 5%-", false)
          awesome.emit_signal("volume_change")
       end,
       {description = "volume down", group = "hotkeys"}
    ),
    awful.key({}, "XF86AudioMute",
       function()
-         awful.spawn("amixer -D pulse set Master 1+ toggle", false)
+         os.execute("/home/$USER/.config/awesome/components/toggle_mute.sh")
```

Comment:
Can we instead call awful.spawn to run each command in the toggle_mute script?

Reply:
I don't think that should cause any problems, so it's probably good.